specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/ServerAzureADAdministrators.json

```diff
@@ -0,0 +1,463 @@
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "The specified Azure Active Directory administrator does not exist."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Delete Azure Active Directory administrator.": {
+            "$ref": "./examples/AdministratorDelete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/administrators": {
+      "get": {
+        "tags": [
+          "ServerAzureADAdministrators"
+        ],
+        "description": "Gets a list of Azure Active Directory administrators in a server.",
+        "operationId": "ServerAzureADAdministrators_ListByServer",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of Azure Active Directory administrators.",
+            "schema": {
+              "$ref": "#/definitions/AdministratorListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Gets a list of Azure Active Directory administrator.": {
+            "$ref": "./examples/AdministratorList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/disableAzureADOnlyAuthentication": {
```

Comment:
why is there a POST action to disable this? its a property on the resource type, and so can be updated via a PATCH.

Reply:
its not for path. its an action. and had no body, only updates a value implicitly.

Reply:
OK, looks like different RBAC may be needed to allow modifying this property - someone may not have /write access on the resource but they can still update this property.

Reply:
yes. that is the intention here. to keep permissions separate for setting and un-setting that value.